src/services/cgnService.ts

```diff
@@ -21,12 +21,13 @@ import {
 } from "italia-ts-commons/lib/responses";
 
 import { fromNullable } from "fp-ts/lib/Option";
-import { EycaActivationDetail } from "generated/io-cgn-api/EycaActivationDetail";
-import { EycaCard } from "generated/io-cgn-api/EycaCard";
-import { InstanceId } from "../../generated/io-cgn-api/InstanceId";
-import { CgnActivationDetail } from "../../generated/io-cgn-api/CgnActivationDetail";
+import { EycaActivationDetail } from "generated/cgn/EycaActivationDetail";
+import { EycaCard } from "generated/cgn/EycaCard";
+import { InstanceId } from "../../generated/cgn/InstanceId";
+import { CgnActivationDetail } from "../../generated/cgn/CgnActivationDetail";
```

Comment:
What is the approach we want to use? We use on other services the refs from the API (mostly) into other services we use the reference from the backend. Here we are changing the approach.

Reply:
You're right I will change :)